PICMI_Python/particles.py

```diff
@@ -282,6 +282,80 @@ def __init__(self, density,
         self.handle_init(kw)
 
 
+class PICMI_FoilDistribution(_ClassWithInit):
+    """
+    Describes a foil with optional exponential pre- and post-plasma ramps along the propagation direction.
+
+    Parameters
+    ----------
+    density: float
+        Physical number density [m^-3]
+
+    front : float
+        y-postion of front surface of foil [m]
+
+    thickness: float > 0
```

Comment:
Could this be zero so that there would be only the pre- and post-plasma ramps?

Reply:
thickness and pre/post-plasma cutoffs can be zero ( I will update the code to reflect this), but not pre/post-plasma lengths as that would cause divide by zero error in density ~ exp(y/l) ==> l>0